.config/requirements-testinfra.txt

```
@@ -0,0 +1 @@
+pytest-testinfra==8.1.0
```

Could you please change `==` to `>=`?
```suggestion
pytest-testinfra>=8.1.0
```

Why `pytest-testinfra==8.1.0` is used in `requirements.txt`?
It's not an optional dependency. If we take a deep look at `requirements.txt`, it cannot be installed by any `pip install molecule[xxx]`.

---

So, I removed it from `requirements.txt` and made it as an optional dependency which will allow it to be installed using `pip install molecule[testinfra]`